.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
+
```

Should have minimal permissions. I.e.; 

```suggestion
permissions:
  contents: read
```

---

Thanks - got this from GitHub's starter workflow, but I guess it's not perfect.